cms/models.py

```diff
@@ -35,14 +36,17 @@ def get_context(self, request):
             "host": webpack_dev_server_host(request)
         }
 
+        username = None
+        if not request.user.is_anonymous():
+            username = request.user.social_auth.get(provider=EdxOrgOAuth2.name).uid
```

Comment:
well, I guess the staff users need at least to be able to work in the CMS and the django admin. We can open another issue to handle the dashboard.